<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Ground segmentation -->
  <node name="ground_segmentation" pkg="ground_segmentation" type="ground_segmentation_node" output="screen">
    <rosparam command="load" file="$(find ground_segmentation)/launch/segmentation_params.yaml"/>
    <param name="input_topic" value="/velodyne_points" />
    <param name="ground_output_topic" value="/ground_cloud" />
    <param name="obstacle_output_topic" value="/obstacle_cloud" />
  </node>

</launch>